ElectronicObserver/ViewModels/FormMainViewModel.cs

```diff
@@ -877,6 +884,20 @@ private void OpenQuestTrackerManager()
 
 		new QuestTrackerManagerWindow().Show(Window);
 	}
+	
+	[ICommand]
+	private void OpenEquipmentUpgradePlanner()
+	{
+		if (!KCDatabase.Instance.Equipments.Any())
```

Comment:
Might be better to extract this into ToolService.

Reply:
Didn't know about ToolService, should be fine now